app/models/embed/purl.rb

```diff
@@ -69,10 +74,22 @@ def downloadable_files
       end
     end
 
+    def all_files_downloadable?
```

Comment:
Is this method needed?  I'm not seeing where it is used.

Reply:
Ah good catch, this is no longer used/needed after https://github.com/sul-dlss/stacks/pull/950